src/components/Card/index.tsx

```diff
@@ -203,11 +206,23 @@ const TopLeftAdornment = styled.div`
   }
 `
 
+const Period = styled.div`
+  display: inline-flex;
+  align-items: center;
+`
+
+const PeriodText = styled(TinyText)`
+  padding-inline-start: ${space[4]};
+
+  :first-letter {
+    text-transform: capitalize;
```

Comment:
What is the reason you are doing this?

Reply:
I now do it from the util itself, so not necessary anymore. The auto generated date like today or tmw were not capitalize and i thought first it would be easier to do it from here but at the end not. I removed it